### DISCUSSION TITLE
Why is this happening

### DISCUSSION THREAD

**Discussion by baltazargato** [community/community#28620] [Codespaces]
I tried to do a cs50 assignment that uses vs code and when I tried to go into vs code through Github this popped up. How can I deal with this? I have restarted countless times, logged in and out of my account, and even unenrolled and reenrolled in my course, and no luck.
![Screenshot (31)](https://user-images.githubusercontent.com/108493389/183269806-aa25d48f-5ac9-430e-893f-52f0a0f2b2a2.png)

**Comment by jfullstackdev** [comment]
you may want to check out other posts too about cs50

https://github.com/orgs/community/discussions?discussions_q=cs50

**Reply by baltazargato** [reply]
@jdevstatic I am working on a school-issued computer. Github is not blocked but could their blocks be the cause of this?  If so, is there a way around this?

**Comment by jaclynj** [comment]
Hi @baltazargato are you still having trouble? I can see that your codespace is in a valid state. Something might be blocking codespaces from making a request in web.

1. Can you walk me through the steps you're taking to create a codespace? 
2. Could you let me know if you see any errors in the network tab or in the console besides 'failed to fetch'?
3. Have you tried using the VS Code desktop app? Let me know what happens.

**Reply by baltazargato** [reply]
@jaclynj I tried using the desktop but I went on other discussions that were similar and realized that the problem was, since I was using a school issued laptop, the school's firewall could be blocking something in the sign in process. So I figured out a way around it and it has started working again.

**Reply by jaclynj** [reply]
Nice I'm glad you fixed it! What did you do, in case someone has a similar problem?

**Reply by baltazargato** [reply]
I downloaded a VPN. Most of them were blocked but I found one that wasn't. Most school firewalls are different, but this seemed to work for me as it took off any website restrictions.

**Reply by jfullstackdev** [reply]
yes, that's a very good one, others might have the same situation with cs50, the other similar post is, the antivirus is now blocking this

**Reply by legomushroom** [reply]
@baltazargato @jdevstatic there should be a blocked request log in the Console tab or you can filter to blocked requests in the Network tab in the Dev Tools.

<img width="1157" alt="image" src="https://user-images.githubusercontent.com/1478800/183725219-05763250-b88d-43fa-85d0-63136241bdc0.png">

Do you see what is the request that is being blocked? If you cannot find it, please send the HAR file for the page load? You can export it on the Network tab in the Dev Tools 🙏 

If you need to send anything that might contain personal info, you can DM it to me directly at legomushroom@github.com instead of posting it here 🔐
